main.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+"""
+Entry point to run the workflow.
+"""
+import os
+
+import coloredlogs
+import click
+from cpg_utils.workflows.workflow import get_workflow
+from cpg_utils.config import set_config_paths
+from stages.seqr_loader import MtToEs
+
+fmt = '%(asctime)s %(levelname)s (%(name)s %(lineno)s): %(message)s'
+coloredlogs.install(level='DEBUG', fmt=fmt)
+
+
+@click.command()
+@click.argument('config_paths', nargs=-1)
+def main(config_paths: list[str]):
+    """
+    Run a workflow, using CONFIG_PATHS in the order specified, overriding
+    $CPG_CONFIG_PATH if specified.
+    """
+    if _cpg_config_path_env_var := os.environ.get('CPG_CONFIG_PATH'):
+        config_paths = _cpg_config_path_env_var.split(',') + config_paths
+    set_config_paths(list(config_paths))
```

Comment:
Is this meant for tests only? I'm asking because I thought we'd typically want users to rely on the `--config` parameter in the analysis-runner to invoke workflows.